agents/vmware_soap/fence_vmware_soap.py

```diff
@@ -57,7 +57,8 @@ def soap_login(options):
 
 	try:
 		headers = {"Content-Type" : "text/xml;charset=UTF-8", "SOAPAction" : "vim25"}
-		conn = Client(url + "/vimService.wsdl", location=url, transport=RequestsTransport(verify=verify), headers=headers)
+		login_timeout = 60 if "--disable-timeout" in options and options["--disable-timeout"] != "false" else int(options["--login-timeout"])
```

Comment:
It's easier to just do `or 60` like here:
https://github.com/ClusterLabs/fence-agents/blob/master/agents/gce/fence_gce.py#L296

As disable-timeout can be other true/false values as well.

Reply:
Ok, I now understand. My bad.

Reply:
No worries :).